### DISCUSSION TITLE
Is there an option in pyright to not use stubs in typeshed?

### DISCUSSION THREAD

**Discussion by tasuren** [microsoft/pyright#5538] [Q&A] [answered]
I am currently developing a Python project using Redis and there is an error in the stub file of the redis-py redis wrapper for Python in typeshed.
There are several errors, but I don't currently have the time to do a pull request or anything like that, so I would like to use the annotations in the redis-py code as the types instead of stub file for now.
In other words, I don't want to use the stub of typeshed, I want to use the annotations in the code for type checking.
Can this be done?

**Comment by erictraut** [comment, marked as answer]
You could place a "py.typed" marker in the `redis-py` directory within your `site-packages`. This will cause pyright to ignore the typeshed stubs and use the `redis-py` instead. Alternatively, you could copy the typeshed stubs for `redis-py` into a project-local `typings` directory and make local edits to it as you see fit. This will give you a private set of stubs that override the public typeshed version.

**Reply by tasuren** [reply]
I see. Thank you for your answer.
